docs/tutorials/point_source.ipynb

```diff
@@ -199,54 +185,32 @@
-    "    else:\n",
-    "        ax[0].plot(src[\"x\"], src[\"y\"], \"bx\", mew=2)\n",
-    "\n",
-    "# Display the model\n",
-    "model = blend.get_model()\n",
-    "model_ = observation.render(model)\n",
-    "img_rgb = scarlet.display.img_to_rgb(model_, norm=norm)\n",
-    "ax[1].imshow(img_rgb)\n",
-    "ax[1].set_title(\"Model\")\n",
-    "for k, src in enumerate(blend.sources):\n",
-    "    y,x = src.pixel_center\n",
-    "    ax[1].text(x, y, str(k), color=\"r\")\n",
-    "\n",
-    "# Display the residual\n",
-    "residual = images-model_\n",
-    "img_rgb = scarlet.display.img_to_rgb(residual)\n",
-    "ax[2].imshow(img_rgb)\n",
-    "ax[2].set_title(\"Residual\")\n",
-    "\n",
-    "# Show the plots\n",
-    "plt.show()\n",
-    "\n",
-    "# Show the morphologies\n",
-    "for k, src in enumerate(blend.sources):\n",
-    "    asinh = AsinhMapping(minimum=0, stretch=1/20, Q=10)\n",
-    "    morph = scarlet.display.img_to_rgb(src.morph, norm=asinh)\n",
-    "    plt.title(\"Source {0}: star={1}\".format(k, catalog[catalog[\"index\"]==k][0][\"is_star\"]))\n",
-    "    plt.imshow(morph)\n",
-    "    plt.show()\n",
-    "\n",
-    "print(\"Total residual {0}\".format(np.sum(np.abs(residual))))"
+    "scarlet.display.show_scene(sources, \n",
+    "                           norm=norm, \n",
+    "                           observation=observation, \n",
+    "                           show_rendered=True, \n",
+    "                           show_observed=True, \n",
+    "                           show_residual=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scarlet.display.show_sources(sources, \n",
+    "                             norm=norm, \n",
+    "                             observation=observation,\n",
+    "                             show_rendered=True, \n",
+    "                             show_observed=True)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The dipoles we see at the center of the stellar sources is because the catalog offset the centers of all the sources by a random amount and by fixing the morphology we also cannot adjust the position, which also caused the code to require a greater number of iterations for convergence. A future version of *scarlet* will hopefully address this unfortunate side effect of fitting PSF stars."
+    "We can see that the model overall performs well, especially in fitting the amplitude and locations of the point sources. It is worth noting that the stellar residuals have red ring and blue core, which suggest that the PSF model of the observation is not quite correct across all bands. The extended sources are less convincing. The effects of the symmetry constraint are clearly visible, and the color between the two galaxies is not very different, so these two sources remain poorly separated."
```

Comment:
Yes, but it is true that this tutorial looks worse now than it did in the master branch, and I don't understand why.

Reply:
Did you try using a wider model kernel? If I remember correctly this simulated image had a very wide PSF, so it might be that the difference kernel is just too narrow and the code doesn't calculate the  difference kernel properly, so it has trouble going from the model PSF to the observed PSF. The current tutorial used `sigma=0.9` for example.

I'm also not sure that the current version is any better. The residuals are sooo much larger in the current master because of the dipoles on the stellar sources that it's hard to tease out if there were also issues with the PSF matching.

Reply:
I just tried: no change. I will claim the PSF model is off. However, it's the extended galaxies that look pretty ratty now. It's unclear whether we now notice PSF errors much more strongly... You can also see the effect of the box limitation in source 4.

Reply:
I'm surprised that the PSF model might be off. It's a simulated image using galsim, with the PSF galsim used to generate the image. Could it have something to do with the way noise was added to the image?

Reply:
No. Are you sure that the PSF model image has pixel integration? @herjy has had some trouble with galsim as well. We make the assumptions that all "images" are pixel-integrated, otherwise the diff kernels are incorrect.

Reply:
I use `psf.drawImage(nx=31,ny=31, scale=1).array` with a galsim `Gaussian` PSF to generate the psf image. Looking at the [galsim docs](http://galsim-developers.github.io/GalSim/_build/html/gsobject.html#galsim.GSObject.drawImage) for `drawImage` it looks like this should be integrated by default.

Reply:
This is a problem for the docs, not the code.